ui/keyboardMove/src/plugins/keyboardMove.test.ts

```diff
@@ -79,6 +79,23 @@ describe('keyboardMove', () => {
     expect(input.value).toBe('');
   });
 
+  test('next command', () => {
```

Comment:
Nitpick, but typically you name the test so it's more like spoken English. Something like
```typescript
test('goes to next puzzle', () => {
```
is nice because if someone is reading the test output, they more readily know what should be going on.

Reply:
yeah, I tried to keep it generic but given that for now the only implementation will be to move to the next puzzle I might as well.